publiccode.yml

```diff
@@ -43,32 +43,26 @@ description:
     videos:
       - 'https://www.youtube.com/watch?v=QWt6vB-cipE'
 developmentStatus: beta
-landingURL: 'https://standard.publiccode.net/'
+landingURL: 'https://standard-for-public-code.github.io/standard-for-public-code/'
 legal:
-  authorsFile: 'https://raw.githubusercontent.com/publiccodenet/standard/master/AUTHORS.md'
+  authorsFile: 'https://raw.githubusercontent.com/standard-for-public-code/standard-for-public-code/master/AUTHORS.md'
   license: CC0-1.0
   mainCopyrightOwner: Foundation for Public Code
```

Comment:
I think I saw this replaced with "the contributors" or something elsewhere?

Reply:
This comment is about line 50 "mainCopyrightOwner"

Reply:
```suggestion
  mainCopyrightOwner: The contributors
```